### PR TITLE
[sciter] Update to 4.4.3.20.7852

### PR DESCRIPTION
**Pull request by pravic**

### CODE REVIEW

File: ports/sciter/CONTROL [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Source: sciter
-Version: 4.4.3.18
+Version: 4.4.3.20-1
 Homepage: https://github.com/c-smile/sciter-sdk
 Description: Sciter is an embeddable HTML/CSS/scripting engine.
 Supports: !uwp
+
+Feature: windowless
+Description: Uses Sciter.Lite builds
```
Comment (PhoebeHui) [inline ports/sciter/CONTROL:8]: Have you test the feature locally? The CI system doesn't test features.

**Review by PhoebeHui — COMMENTED** [review]
Thanks for the PR!

File: ports/sciter/CONTROL [hunk 2]
```diff
@@ -1,5 +1,8 @@
 Source: sciter
-Version: 4.4.3.18
+Version: 4.4.3.20-1
```
Comment (PhoebeHui) [inline ports/sciter/CONTROL:2]:
```suggestion
Version: 4.4.3.20
```

**Review by PhoebeHui — COMMENTED** [review]
I updated the Version to '4.4.3.20'

File: ports/sciter/CONTROL (hunk 1 above)
Comment (pravic) [reply ports/sciter/CONTROL:8]: I have tested this, yes.

File: ports/sciter/CONTROL (hunk 2 above)
Comment (pravic) [reply ports/sciter/CONTROL]: To be fair, 4.4.3.20 means the original library version. 20-1 means that a vcpkg port maintainer has modified it: https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**